test/Map.test.mo

```diff
@@ -594,7 +594,7 @@ run(
           let map = Map.singleton<Nat, Text>(1, "1");
           Map.toText<Nat, Text>(map, Nat.toText, func(value) { value })
         },
-        M.equals(T.text("(1, 1)"))
+        M.equals(T.text("{(1, 1)}"))
```

Comment:
This is what people coming from JS / Python would expect from the `toText()` function:

```suggestion
        M.equals(T.text("{ 1 = 1 }"))
```

Reply:
Right, but is that because it's valid syntax for constructing  map in JS/Python?

Reply:
Ideally, I'd like to be able to cut-n-paste the toText output into some Motoko code to reconstruct the value, but perhaps that's just unrealistic.